Library/Formula/emacs.rb

```diff
@@ -26,6 +26,7 @@ class Emacs < Formula
 
   option "with-cocoa", "Build a Cocoa version of emacs"
   option "with-ctags", "Don't remove the ctags executable that emacs provides"
+  option "with-libxml2", "Build with libxml2 support"
```

Comment:
Just involves changing the option to `without-libxml2` rather than `with-libxml2` if that's unclear at all.